biit_server/app.py

```diff
@@ -1,23 +1,69 @@
-from flask import Flask
+from flask import Flask, request
+import json
 
 
 # This runs on Firebase/Cloud Run!
+def create_app():
+    app = Flask(__name__)
 
-app = Flask(__name__)
+    def missing_data(missedParam, response):
+        return "Missing Parameter " + missedParam, response
 
+    @app.route("/test")
+    def test_route():
+        return "Ok!"
 
-@app.route("/test")
-def test_route():
-    return "Ok!"
+    @app.route("/account", methods=["POST", "GET", "PUT", "DELETE"])
+    def account_route():
+        if request.method == "POST":
+            data = ""
+            try:
+                data = json.loads(request.get_json())
+            except:
+                return "Failed to load account JSON", 400
+            if "token" not in data or not data["token"]:
+                return missing_data("token", 400)
+            if "fname" not in data or not data["fname"]:
+                return missing_data("fname", 400)
+            if "lname" not in data or not data["lname"]:
+                return missing_data("lname", 400)
+            if "email" not in data or not data["email"]:
+                return missing_data("email", 400)
```

Comment:
is there a way that we can make this iterate over a collection of fields, so if we need to modify or change things in the future less code is required?